lib/vrf.c

```diff
@@ -1000,8 +1000,7 @@ lib_vrf_state_active_get_elem(struct nb_cb_get_elem_args *args)
 	struct vrf *vrfp = (struct vrf *)args->list_entry;
 
 	if (vrfp->status == VRF_ACTIVE)
-		return yang_data_new_bool(
-			args->xpath, vrfp->status == VRF_ACTIVE ? true : false);
+		return yang_data_new_bool(args->xpath, true);
```

Comment:
hmm - I thought we were trying to include the booleans in json even if "false" ?

Reply:
oh, sorry - I read "json" instead of "yang".
you're quite right - no need to include the boolean.